iset.mm

```diff
@@ -95229,6 +94988,217 @@ gives an ordered pair whose first element is the index (translated
+        UDTZUWTVNTZUOZUURTZUWGUXEUWTUXHUURUXDUWTUXHUEUVDUWTUULGWLWMUKUXEUXIUXFU
+        TVAVBZUXFUXGIVBZUOZUWGUXEUXIUXFUXGUURVBZUXLUXFUXGUURWNUXEUXFUULSZUXGHSU
+        XLUWGSZUXMUXLUEUXDUXNUVDUWTUULGWOWMZUXEGHUXGAGHXRZUVBYRUXDNWPUXDUXGGSUV
+        DUWTUULGWKWMZWQUXEUXJUULSZUXKGSZUXOUXEUXNUXSUXPEUXFWIVJUXEUXFUUOIVBZGSZ
+        UXTCGUXGUUOUXGUEZUYAUXKGUUOUXGUXFIWBZWCUXEUWBUYBCGWDBUULUXFUUMUXFUEZUWA
+        UYBCGUYEUUPUYAGUUMUXFUUOIWEZWCWFAUWEUVBYRUXDUWFWPUXPWHUXRWHUXJUXKUULGWS
+        URZBCUXFUXGUULHUUQUXLUURUXJUYAUOUWGUYEUUNUXJUUPUYAUUMUXFUTVAWEUYFWTUYCU
+        YAUXKUXJUYDXHUURXAZXBXCXDUYGXEXEXFAUXCUVBYRAEUULSZUUKUXCAUUJUYILEXGVJME
+        DUULGWSURVRUVNUCUUHYNUWGUURXIXCYSFUUSPVEYOUWRUURYNFUUSPVEXJXKUVDYOUWNUU
+        RUVDUWHYOUWNUEUWIYOUULGWLVJUKXLUVDUWNUWLUURUVDYPYQUVFUVCYRXMXSUKXLUVDUW
+        MYQUVFUURVBZUVHYQUVFUURWNUVDUVLUVFHSUVHUWGSZUYJUVHUEUVOUVDGHUVFAUXQUVBY
+        RNVRUWJWQUVDUVJUVKUYKUVPUWKUVEUVGUULGWSURBCYQUVFUULHUUQUVHUURUVEUVQUOUW
+        GUWCUUNUVEUUPUVQUUMYQUTVAWEUWDWTUVSUVQUVGUVEUVTXHUYHXBXCXDXLUKUVDBYNEJA
+        UUJUVBYRLVRRUVNXNVMXOXTXPXQ $.
+    $}
+
+    ${
+      $d C v x y $.  $d C x y $.  $d F x y $.  $d R v $.  $d R x y $.
+      $d G x y $.  $d S x y $.  $d T x y $.  $d ph v x y $.  $d ph x y $.
+      frecuzrdgdom.2 $e |- G = frec ( ( x e. ZZ |-> ( x + 1 ) ) , C ) $.
+      $( The domain of the result of the recursive definition generator on
+         upper integers.  (Contributed by Jim Kingdon, 24-Apr-2022.) $)
+      frecuzrdgdom $p |- ( ph -> dom ran R = ( ZZ>= ` C ) ) $=
+        ( cfv com syl wcel vv crn cdm cuz cxp wss frecuzrdgrclt frn dmss dmxpss
+        wf syl6ss ccnv c2nd cop c1st wceq adantr wfun ffun frec2uzf1od f1ocnvdm
+        cv wa sylan fdm eleqtrrd fvelrn syl2anc sseldd 1st2nd2 cz co frecuzrdgg
+        wf1o adantlr f1ocnvfv2 eqtrd opeq1d eqeltrrd wi simpr xp2nd opeldmg mpd
+        ex ssrdv eqssd ) AFUBZUCZEUDQZAWJWKGUEZUCZWKAWIWLUFZWJWMUFARWLFUKZWNABC
+        DEFGHIKLMNOUGZRWLFUHSZWIWLUISWKGUJULAUAWKWJAUAVCZWKTZWRWJTZAWSVDZWRWRJU
+        MQZFQZUNQZUOZWITZWTXAXCXEWIXAXCXCUPQZXDUOZXEXAXCWLTZXCXHUQXAWIWLXCAWNWS
+        WQURXAFUSZXBFUCZTXCWITAXJWSAWOXJWPRWLFUTSURXAXBRXKARWKJVOZWSXBRTABEJKPV
+        AZRWKWRJVBVEZAXKRUQZWSAWOXOWPRWLFVFSURVGXBFVHVIZVJZXCWKGVKSXAXGWRXDXAXG
+        XBJQZWRXABCDEFGHIJXBAEVLTWSKURADGTWSLURAGHUFWSMURABVCZWKTCVCZGTVDXSXTIV
+        MGTWSNVPOXNPVNAXLWSXRWRUQXMRWKWRJVQVEVRVSVRXPVTXAWSXDGTZXFWTWAAWSWBXAXI
+        YAXQXCWKGWCSWRXDWIWKGWDVIWEWFWGWH $.
+    $}
+
+    ${
+      $d C x y z $.  $d F x y z $.  $d G w x y z $.  $d R v w z $.
+      $d R w x y z $.  $d S x y z $.  $d T x y z $.  $d ph v w z $.
+      $d ph w x y z $.
+      frecuzrdgfun.g $e |- G = frec ( ( x e. ZZ |-> ( x + 1 ) ) , C ) $.
+      $( The recursive definition generator on upper integers produces a a
+         function.  (Contributed by Jim Kingdon, 24-Apr-2022.) $)
+      frecuzrdgfun $p |- ( ph -> Fun ran R ) $=
```

Comment:
This theorem does not depend on G!? Why is frecuzrdgfun.g needed?

Reply:
As with the previous comment, a lot of the steps in this proof have a `G` in them, so this is to make the proof more readable.